src/styles/scss/index.scss

```diff
@@ -1,4 +1,8 @@
-@import 'base/variables';
-@import 'base/elements';
+@import "base/variables";
```

Comment:
I think we need to decide whether we go with double/single quotes, for consistency.

Reply:
I (and I think most - airbnb rule) use single for JS, double for JSX, as this retains consistency with HTML.